_includes/nav.html

```diff
@@ -1,87 +1,147 @@
+                      >About us</a
+                    >
+                  </li>
+                  <li>
+                    <a
+                      class="dropdown-item"
+                      href="{{ site.baseurl }}/about-us/#mission"
+                      >Our mission</a
+                    >
+                  </li>
+                  <li>
+                    <a
+                      class="dropdown-item"
+                      href="{{ site.baseurl }}/leadership"
+                      >Leadership and governance</a
+                    >
+                  </li>
+                  <li>
+                    <a
+                      class="dropdown-item"
+                      href="{{ site.baseurl }}/contact-us"
+                      >Contact us</a
+                    >
+                  </li>
                 </ul>
-	      <li class="nav-item"><a class="nav-link" href="{{ site.baseurl }}/#try">Try now</a></li>
-	      <li class="nav-item"><a class="nav-link" href="{{ site.baseurl }}/develop">Join development</a></li>
-	      <li class="nav-item"><a class="nav-link" href="{{ site.baseurl }}/volunteering">Volunteer</a></li>
-	      <li class="nav-item"><a class="nav-link" href="{{ site.baseurl }}/press">News</a></li>
-	      <li class="nav-item"><a class="nav-link" href="{{ site.baseurl }}/donate">Donate</a></li>	  
+              </li>
+
+              <li class="nav-item">
+                <a class="nav-link" href="{{ site.baseurl }}/#try">Try now</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="{{ site.baseurl }}/develop"
+                  >Join development</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="{{ site.baseurl }}/volunteering"
+                  >Volunteer</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="{{ site.baseurl }}/press">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="{{ site.baseurl }}/donate">Donate</a>
+              </li>
             </ul>
-	  </div>
-	</div>
+          </div>
+        </div>
       </nav>
     </div>
   </div>
 </header>
 
 <!-- Header for screens with width <= 1215px -->
-<div class="container" id="secondHeader" >
-  <div class="row">
-    <div class="col-md-12">
-      <div class="wrapper">
-        <nav id="sidebar">
-          <div id="dismiss">
-            <i class="ion-arrow-left-a"></i>
-          </div>
-          <div class="sidebar-header">
-            <h3>Sugar Labs</h3>
+<div class="sticky-top bg-white container-fluid flex-fill w-100" id="secondHeader">
+  <nav class="navbar navbar-expand-lg navbar-light">
+    <a class="navbar-brand" href="{{ site.baseurl }}/">
+      <img src="{{ site.baseurl }}/assets/logoOpt.png" alt="Sugar Labs logo" height="80" width="240">
```

Comment:
If possible, it would be nice to have the random logo color feature here as well.